app/graphql/mutations/create_offer_response_mutation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateOfferResponseMutation < Mutations::BaseMutation
+    argument :offer_id, ID, required: true
+    argument :intended_state, Types::IntendedStateType, required: true
+
+    argument :phone_number, String, required: false
+    argument :comments, String, required: false
+    argument :rejection_reason, String, required: false
+
+    field :consignment_offer_response, Types::OfferResponseType, null: true
```

Comment:
Externally, these types are prefaced with `consignment_` to disambiguate from BNMO offers (I'm guessing! I was also following convention here).

Reply:
Naming is hard! 😝